apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/NoAlerts/NoAlerts.tsx

```diff
@@ -0,0 +1,37 @@
+import { NoCasesSvg } from '@/common/components/atoms/icons';
```

Comment:
Can we combine NoCases and NoAlerts into 1 component?

Reply:
Adding as a tech debt task.